MonoGame.Framework.DesktopGL.sln

```diff
@@ -41,6 +43,18 @@ Global
 		{3EA1BB98-6F7F-4CEC-9804-8553A599AA3B}.Release|x64.Build.0 = Release|Any CPU
 		{3EA1BB98-6F7F-4CEC-9804-8553A599AA3B}.Release|x86.ActiveCfg = Release|Any CPU
 		{3EA1BB98-6F7F-4CEC-9804-8553A599AA3B}.Release|x86.Build.0 = Release|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Debug|x64.Build.0 = Debug|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Debug|x86.Build.0 = Debug|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Release|Any CPU.Build.0 = Release|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Release|x64.ActiveCfg = Release|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Release|x64.Build.0 = Release|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Release|x86.ActiveCfg = Release|Any CPU
+		{462504E8-562E-4A21-A34C-8990A10681E7}.Release|x86.Build.0 = Release|Any CPU

```

Comment:
@Jjagg @harry-cpp 

Why do we have x86 and x64 targets at all?  Shouldn't we just have Any CPU?